include/RuntimeLib.lua

```diff
@@ -10,6 +10,27 @@ local TS = {}
 
 TS.Promise = Promise
 
+local Symbol = {}
+Symbol.__index = Symbol
+
+function Symbol.new(description)
+	local self = setmetatable({}, Symbol)
+	self.description = description
+	return self
+end
+
+function Symbol:__tostring()
+	return "Symbol(" .. self.description .. ")"
+end
+
+Symbol.hasInstance = Symbol.new("hasInstance")
```

Comment:
It's confusing to add this without adding the related `instanceof` support at the same time. It could be a separate PR that adds the `.hasInstance` assignment.